packages/api/server/models/statsModel/getStats.js

```diff
@@ -62,19 +62,28 @@ module.exports = async (user, location) => {
             type: sequelize.QueryTypes.SELECT,
         },
     );
-    const users = await sequelize.query(
-        `SELECT 
-            u.created_at
-        FROM users u
-        LEFT JOIN localized_organizations lo ON u.fk_organization = lo.organization_id
-        WHERE u.fk_status = 'active'
+    const connectedUsers = await sequelize.query(
+        `SELECT
+                unl.fk_user AS user_id,
+                TO_CHAR(unl.datetime, 'YYYY-MM-DD') AS date_log
+        FROM
+                user_navigation_logs unl 
+        LEFT JOIN
+                users u ON u.user_id = unl.fk_user
+        LEFT JOIN
+                localized_organizations lo ON u.fk_organization = lo.organization_id
         ${where}
-        ORDER BY u.created_at DESC`,
+        GROUP BY
+                date_log,
+                fk_user
+        ORDER BY
+                date_log DESC,
+                fk_user`,
```

Comment:
Pour faciliter l'exploitation derrrière, tu peux probablement générer le total par semaine plus facilement ici dans la requête SQL que dans `decomposeForDiagram`.
Ca donne quelque chose de ce genre :
```
SELECT
  COUNT(DISTINCT fk_user),
  CONCAT(EXTRACT(week from datetime), '/', EXTRACT(year from datetime)) AS week
FROM user_navigation_logs
GROUP BY CONCAT(EXTRACT(week from datetime), '/', EXTRACT(year from datetime))
ORDER BY CONCAT(EXTRACT(week from datetime), '/', EXTRACT(year from datetime)) ASC;
```

Reply:
À voir avec JP s'il faut compter les admins nationaux dans ce total ou pas (si on s'aligne sur nos WAU affichés, il faudrait les ignorer : `WHERE users.fk_role IS NULL OR users.fk_role <> 'national_admin'`

Reply:
Intéressant. Mais l'affichage dans la vue d'ensemble est fait non pas par semaine calendaire mais par semaine glissante (au sens de 7 jours consécutifs). Chaque semaine de référence pour l'affichage des composants `bar` est calculée depuis le jour courant. Voir [cette méthode](https://github.com/MTES-MCT/resorption-bidonvilles/blob/develop/packages/api/server/models/statsModel/_common/getArrayOfDates.js).